README.md

```diff
@@ -3,8 +3,6 @@
 
 ### The EuroPi is a fully user reprogrammable module based on the [Raspberry Pi Pico](https://www.raspberrypi.com/products/raspberry-pi-pico/), which allows users to process inputs and controls to produce outputs based on code written in Python. The entire project is open-source.
```

Comment:
Any interest in dropping the bolding here? Otherwise LGTM.

Reply:
Yeah I agree with that. Done.